dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05200WorkflowTaskUniqueKey.java

```diff
@@ -40,22 +41,36 @@ public void executeUpgrade() throws DotDataException, DotRuntimeException {
         } catch (SQLException e) {
             throw new DotDataException(e.getMessage(), e);
         }
+
         final DotConnect dotConnect = new DotConnect();
-        
+
+        // 18524: set to default language records with null language id
+        try {
+            final long defaultLangId = dotConnect.setSQL("select id from language").loadLong("id");
+            dotConnect.executeStatement(String.format(
+                    "update workflow_task set language_id='%s', mod_date=%s where language_id is null",
+                    defaultLangId,
+                    getDbDate()));
```

Comment:
Let's send the date from dotcms side. Use `addParam(new Date())`

Reply:
Using Date() now.